crates/cli/tests/reference/echo.d.ts

```diff
@@ -0,0 +1,185 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_u8(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_i8(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_u16(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_i16(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_u32(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_i32(a: number): number;
+/**
+ * @param {bigint} a
+ * @returns {bigint}
+ */
+export function echo_u64(a: bigint): bigint;
+/**
+ * @param {bigint} a
+ * @returns {bigint}
+ */
+export function echo_i64(a: bigint): bigint;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_usize(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_isize(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_f32(a: number): number;
+/**
+ * @param {number} a
+ * @returns {number}
+ */
+export function echo_f64(a: number): number;
+/**
+ * @param {boolean} a
+ * @returns {boolean}
+ */
+export function echo_bool(a: boolean): boolean;
+/**
+ * @param {string} a
+ * @returns {string}
+ */
+export function echo_char(a: string): string;
+/**
+ * @param {string} a
+ * @returns {string}
+ */
+export function echo_string(a: string): string;
+/**
+ * @param {Uint8Array} a
+ * @returns {Uint8Array}
+ */
+export function echo_vec_u8(a: Uint8Array): Uint8Array;
+/**
+ * @param {Foo} a
+ * @returns {Foo}
+ */
+export function echo_struct(a: Foo): Foo;
+/**
+ * @param {(Foo)[]} a
+ * @returns {(Foo)[]}
+ */
+export function echo_vec_struct(a: (Foo)[]): (Foo)[];
+/**
+ * @param {number | undefined | null} [a]
+ * @returns {number | undefined}
+ */
+export function echo_option_u8(a?: number): number | undefined;
```

Comment:
Something happened in the last rebase and the `| null` is missing.

Reply:
Oh god, thanks for catching this!